toolchain/sem_ir/ids.h

```diff
@@ -86,6 +86,20 @@ constexpr InstId InstId::Invalid = InstId(InvalidIndex);
       InstId::ForBuiltin(BuiltinInstKind::Name);
 #include "toolchain/sem_ir/builtin_inst_kind.def"
 
+// An ID of an instruction that is referenced absolutely within a typed
+// instruction. This means that the instruction always represents the ID of a
+// global entity that is independent of the current context, and operations like
+// substitution into the enclosing instruction should not substitute into fields
+// with this type.
+class AbsoluteInstId : public InstId {
```

Comment:
This means we inherit the base class `Print`, but might we want the difference to be visible in the printed output?

Reply:
I don't think we do; we're trying to attach an extra bit of information onto the typed inst field, not change how the field behaves.